templates/mask_generation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>Some more handholding on these variables would be good, especially the ones where we have a default name. If someone reads only the documentation above what columns their data will need, it might not be obvious how that maps to here. </p><p> </p><p>Also, I think the notebook should default to displaying the FOV, and we should have it set to only a single FOV</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1052/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>